### PR TITLE
chore: staking deployment scripts

### PR DESCRIPTION
**Pull request by kupermind**
- Staking deployment scripts.

### CODE REVIEW

File: docs/deployment_v1.2.md [hunk 1]
```diff
@@ -2,17 +2,17 @@ Steps for deploying the tokenomics version 1.2 contracts are as follows:
...
 1. EOA to deploy Tokenomics implementation (`TokenomicsThree`);
 2. TokenomicsProxy to change Tokenomics implementation calling `changeTokenomicsImplementation(TokenomicsThree)`;
-3. EOA to deploy Dispenser (`DispenserTwo`) with VoteWeighting contract being deployed before that in `autonolas-governance`;
-4. EOA to change Dispenser address in VoteWeighting calling `changeDispenser(DispenserTwo)`;
-5. EOA to add a retainer address as a nominee in VoteWeighting;
-6. EOA to deploy staking bridging contracts with StakingFactory contract being deployed before that in `autonolas-registries`;
-7. EOA to set up correct L1->L2 links for all the bridging contracts calling `setL2TargetDispenser(L2 corresponding contract)`;
+3. EOA to deploy Dispenser with VoteWeighting contract being deployed before that in `autonolas-governance`;
+4. EOA to change Dispenser address in VoteWeighting calling `changeDispenser(Dispenser)`;
+5. DAO to unpause staking incentives in Dispenser calling `setPauseState(0)`;
```
Comment (DavidMinarsch) [inline docs/deployment_v1.2.md:7]:
```suggestion
5. EOA to unpause staking incentives in Dispenser calling `setPauseState(0)`;
```

**Review by 77ph — DISMISSED** [review]

File: docs/deployment_v1.2.md (hunk 1 above)
Comment (DavidMinarsch) [reply docs/deployment_v1.2.md:7]: https://etherscan.io/address/0x5e13E07006D68B097DCb5979ef5545C4f9156162

Comment (DavidMinarsch) [reply docs/deployment_v1.2.md:7]: https://etherscan.io/address/0xEEf2c02cFC445bFFf1c891D486424Ef3aBc66AB0

Comment (DavidMinarsch) [reply docs/deployment_v1.2.md:7]: https://etherscan.io/address/0xAB787ceebe8F2b8dca44e3bB744D6e63003ee978

Comment (DavidMinarsch) [reply docs/deployment_v1.2.md:7]: https://etherscan.io/address/0x223902b6C583f18E8dc84AF4E6a8fa523d088B78

Comment (DavidMinarsch) [reply docs/deployment_v1.2.md:7]: https://etherscan.io/address/0x4Ce43b82EF1298E1466D3b3Cee67BA6680bF82FB

Comment (DavidMinarsch) [reply docs/deployment_v1.2.md:7]: https://etherscan.io/address/0x42BA1BbD797E4C90F386F03a544F3b62C1f917d0

Comment (DavidMinarsch) [reply docs/deployment_v1.2.md:7]: https://etherscan.io/address/0x15CD7fAeE048c7673aB818C9e582630F1a924593

**Comment by DavidMinarsch** [comment]
https://arbiscan.io/address/0x10c5525F77F13b28f42c5626240c001c2D57CAd4

**Comment by DavidMinarsch** [comment]
https://gnosisscan.io/address/0x67722c823010CEb4BED5325fE109196C0f67D053

**Comment by DavidMinarsch** [comment]
https://optimistic.etherscan.io/address/0x04b0007b2aFb398015B76e5f22993a1fddF83644

**Comment by DavidMinarsch** [comment]
https://polygonscan.com/address/0xf76953444C35F1FcE2F6CA1b167173357d3F5C17

**Comment by DavidMinarsch** [comment]
https://basescan.org/address/0xcDdD9D9ABaB36fFa882530D69c73FeE5D4001C2d

**Comment by DavidMinarsch** [comment]
https://celoscan.io/address/0xb4096d181C08DDF75f1A63918cCa0d1023C4e6C7

**Review by DavidMinarsch — DISMISSED** [review]

File: docs/deployment_v1.2.md [hunk 2]
```diff
@@ -2,17 +2,17 @@ Steps for deploying the tokenomics version 1.2 contracts are as follows:
...
 8. EOA to transfer ownership rights of Dispenser to Timelock calling `changeOwner(Timelock)`;
-9. DAO to change Tokenomics managers calling `changeManagers(ZeroAddress, ZeroAddress, DispenserTwo)`;
+9. DAO to change Tokenomics managers calling `changeManagers(ZeroAddress, ZeroAddress, Dispenser)`;
 10. DAO to change staking parameters in Tokenomics calling `changeStakingParams()`;
-11. DAO to change Treasury managers calling `changeManagers(ZeroAddress, ZeroAddress, DispenserTwo)`;
+11. DAO to change Treasury managers calling `changeManagers(ZeroAddress, ZeroAddress, Dispenser)`;
 12. DAO to enable bridge deposit processors in Dispenser calling `setDepositProcessorChainIds()`;
-13. DAO to unpause staking incentives in Dispenser calling `setPauseState(3)`;
+13. EOA to set up correct L1->L2 links for all the bridging contracts calling `setL2TargetDispenser(L2 corresponding contract)`;
 14. EOA to transfer ownership rights of all the L2 bridging contracts to Timelock representation calling `changeOwner(Timelock)`.
```
[the remark this answers is not in the document]
Comment (DavidMinarsch) [reply docs/deployment_v1.2.md:16]: 14 was now done - please confirm all is good there

File: docs/deployment_v1.2.md [hunk 3]
```diff
@@ -2,17 +2,17 @@ Steps for deploying the tokenomics version 1.2 contracts are as follows:
...
-4. EOA to change Dispenser address in VoteWeighting calling `changeDispenser(DispenserTwo)`;
-5. EOA to add a retainer address as a nominee in VoteWeighting;
-6. EOA to deploy staking bridging contracts with StakingFactory contract being deployed before that in `autonolas-registries`;
-7. EOA to set up correct L1->L2 links for all the bridging contracts calling `setL2TargetDispenser(L2 corresponding contract)`;
+3. EOA to deploy Dispenser with VoteWeighting contract being deployed before that in `autonolas-governance`;
+4. EOA to change Dispenser address in VoteWeighting calling `changeDispenser(Dispenser)`;
+5. EOA to unpause staking incentives in Dispenser calling `setPauseState(0)`;
+6. EOA to add a retainer address as a nominee in VoteWeighting;
+7. EOA to deploy staking bridging contracts on L1 and L2 with StakingFactory contract being deployed before that in `autonolas-registries`;
 8. EOA to transfer ownership rights of Dispenser to Timelock calling `changeOwner(Timelock)`;
```
[the remark this answers is not in the document]
Comment (DavidMinarsch) [reply docs/deployment_v1.2.md:10]: For this - please confirm Dispenser is in the correct state to transfer ownership.

File: docs/deployment_v1.2.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kupermind) [reply docs/deployment_v1.2.md:16]: All is good, confirmed by scripts

**Review by DavidMinarsch — APPROVED** [review]

**Review by 77ph — APPROVED** [review]

**Pull request merged**